README.md

```diff
@@ -94,26 +101,129 @@ The guidelines described below are based on:
   ```
   shop
   ├── components
-  │   ├── edit-profile.component.ts
-  │   ├── edit-profile.component.spec.ts
-  │   ├── edit-profile.component.html
-  │   ├── home.component.ts
-  │   ├── home.component.spec.ts
-  │   ├── home.component.html
-  │   ├── register.component.ts
-  │   └── register.component.spec.ts
+  │   ├── edit-profile.component.ts
+  │   ├── edit-profile.component.spec.ts
+  │   ├── edit-profile.component.html
+  │   ├── home.component.ts
+  │   ├── home.component.spec.ts
+  │   ├── home.component.html
+  │   ├── register.component.ts
+  │   └── register.component.spec.ts
   ├── models
-  │   ├── shopping-cart.model.ts
-  │   ├── shopping-item.model.ts
-  │   └── user.model.ts
-  └── services
-      └── checkout.service.ts
+  │   ├── shopping-cart.model.ts
+  │   ├── shopping-item.model.ts
+  │   └── user.model.ts
+  ├── services
+  │   └── checkout.service.ts
+  └── index.ts
   ```
 
   *Why?*: A developer can locate the code, identify what each file represents at a glance, the structure is flat as can be, and there is no repetitive nor redundant names.
 
   *Why?*: When there are a lot of files (15+) locating them is easier with a consistent folder structures and more difficult in flat structures.
 
+* Export and access all public members for given bounded context by its `index.ts` file located in the root directory of the context itself (for more information take a look at [this issue](https://github.com/mgechev/angular2-style-guide/issues/10)).
+
+  *Why?*: Having a facade which exports the public members enforces [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)).
+
+  *Why?*: Exporting all public members using a high-level facade shortens the paths to the individual code units.
+
```

Comment:
I think a small sample of what a index.ts would look like would be usefull for a lot a people.